### ISSUE TITLE
Problema com nome de Autor

### ISSUE THREAD

**Issue by MBaltz** [lohhans/CrawlerTXT#2]
Há a existência de um **problema** quando ocorre a **redução de um nome**. 
Por exemplo:
_"frase frase frase frase frase. Mateus B. Almeida"_

Note que há um ponto após o B. Logo a frase entregada seria:
_"frase frase frase frase frase. Mateus B."_

**Comment by lohhans** [comment]
Sim, percebi isso fora do escopo solucionado, o padrão que pude detectar é uma letra maiúscula logo antes do ponto, exemplo:

_"frase frase frase frase frase. Mateus **B**."_

Porém, cortando a frase sem ele ainda ficaria:

_"frase frase frase frase frase. **Mateus**"_

Então, após apagar a palavra que antecede o ponto, teríamos que apagar também tudo que sobrou até a pontuação anterior, no caso _**Mateus**_.

Como desejado restando apenas a frese:

_"frase frase frase frase frase."_
